ts/features/walletV3/transaction/components/WalletTransactionDetailsList.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import { View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  HSpacer,
+  IOStyles,
+  ListItemTransaction,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { Transaction } from "../../../../types/pagopa";
+import { formatNumberCentsToAmount } from "../../../../utils/stringBuilder";
+import { Dettaglio } from "../../../../../definitions/pagopa/Dettaglio";
+import {
+  cleanTransactionDescription,
+  getTransactionIUV
+} from "../../../../utils/payment";
+
+type Props =
+  | {
+      transaction?: Transaction | null;
+      loading: boolean;
+      onPress: (operationDetails: Dettaglio) => void;
+    }
+  | {
+      transaction?: Transaction | null;
+      loading: true;
+      onPress: (operationDetails: Dettaglio) => void;
+    };
+
+/**
+ * This component renders a list of transaction details which currently is just a single item
+ * TODO: Using the actual information, this component is already arranged to handle a list that will be implemented from the BIZ Event implementation (https://pagopa.atlassian.net/browse/IOBP-440)
+ */
+export const WalletTransactionDetailsList = ({
+  transaction,
+  loading,
+  onPress
+}: Props) => {
+  if (loading) {
+    return <SkeletonTransactionDetailsList />;
+  }
+  if (!transaction) {
+    return <></>;
+  }
+
+  const operationDetails: Dettaglio = {
+    importo: transaction.amount.amount,
+    enteBeneficiario: transaction.merchant,
+    IUV: pipe(getTransactionIUV(transaction.description), O.toUndefined)
+  };
```

Comment:
I believe we should move on from "italian code", so maybe rewriting this `Dettaglio` type either in this PR or in a new one. What do you think?